tmcprototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -372,6 +403,16 @@ def StandByTelescope(self):
                 print(CONST.STR_ERR_MSG, except_occured)
                 self._read_activity_message = CONST.STR_ERR_MSG + str(except_occured)
                 self.dev_logging(CONST.ERR_EXE_STANDBY_CMD, int(tango.LogLevel.LOG_ERROR))
+
+        try:
+            self._csp_master_leaf_proxy.command_inout(CONST.CMD_CSP_MASTER_LEAF_STANDBY, CONST.STR_CSP_CBF_DEV_NAME)
+        except Exception as except_occured:
+            print(CONST.ERR_EXE_STANDBY_CMD, self.CspMasterLeafNodeFQDN[0])
```

Comment:
Consider consolidating multiple print statements in one if applicable. Ideally there is no need of print statements.

Reply:
Done wherever possible. The commit id is: https://github.com/ska-telescope/tmc-prototype/commit/dafa12fbac0e08db8ec3ec703afb99ba45db38c1